project/src/main/kotlin/org/ionproject/core/calendar/sql/CalendarData.kt

```diff
@@ -129,4 +132,17 @@ object CalendarData {
                 and
                 $UID = :$UID
             $GROUP_BY"""
+
+    const val CHECK_IF_CLASS_EXISTS = """
+        select count(*) from $COURSE_TABLE as CO 
```

Comment:
I suggest adding a `limit 1` to stop the counting after the first row.

Reply:
Are there multiple instances of the same course on the same calendar term?

Reply:
![image](https://user-images.githubusercontent.com/47888471/81792293-c7548d80-94ff-11ea-87ec-aab2a0cac263.png)

No